### DISCUSSION TITLE
Waste collection schedule in Poznań (Poland)

### DISCUSSION THREAD

**Discussion by Tom61680** [mampfes/hacs_waste_collection_schedule#2194] [General]
Hello,

it look like Sepan-Remondis no longer publishes (https://sepan.remondis.pl/harmonogram [2024]) schedules for Poznań. Insteed there are available schedules on official web page of Poznań city - https://www.poznan.pl/mim/odpady/harmonogramy.html.

I have adopted sepan_remondispl.py for new source - poznan.py. It works fine for me for now (the page has some bugs - for example "<tr><tr>" sequence insteed of "</tr><tr>". If you like you can use it for next update (I'm not so familiar with GitHub to try to do it myself)

[poznan.py.txt](https://github.com/user-attachments/files/16015819/poznan.py.txt)

Best regards 
Tom

PS
Thanks for all your work

**Comment by 5ila5** [comment]
I modified this a bit and added this with #2195

Thanks for the work
